README.md

```diff
@@ -14,16 +15,16 @@ The official [Stripe][stripe] Go client library.
 Make sure your project is using Go Modules (it will have a `go.mod` file in its
 root if it already is):
 
-``` sh
+```sh
 go mod init
 ```
 
 Then, reference stripe-go in a Go program with `import`:
 
-``` go
+```go
 import (
-	"github.com/stripe/stripe-go/v79"
```

Comment:
do we need to change these back to v79?

Reply:
v80 is the last release so the change makes sense.